scripts/validate-test-results

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env perl
```

Comment:
How about:
```
#!/usr/bin/env perl

# Copyright 2023 SUSE LLC
# SPDX-License-Identifier: GPL-2.0-or-later

use strict;
use warnings;

use Mojo::JSON qw(decode_json);
use Mojo::Path;
use Mojo::File qw(path);
use Test::More;

my ($dir) = @ARGV;

my @results = glob("$dir/result-*.json");

for my $result (@results){
    my $path = Mojo::Path->new($result)->canonicalize;
    my $result = decode_json(path($path)->slurp)->{result};
    is $result, 'ok', sprintf 'Test result [%s] ok', $path->to_string;
}

done_testing;

```

Reply:
This looks good! One question though: is `Test::` available on production mode?

I thought those packages would only be available for testing environments

Reply:
You mean Test::More? It's a core module shipped with perl.

Reply:
Ah TIL! I'll use your script then.